src/Microsoft.AspNetCore.SignalR.Core/HubConnectionHandler.cs

```diff
@@ -202,7 +203,7 @@ HubDispatcher<THub> dispatcher
                     {
                         connection.ResetClientTimeout();
 
-                        while (protocol.TryParseMessage(ref buffer, _dispatcher, out var message))
+                        while (!abortedToken.IsCancellationRequested && protocol.TryParseMessage(ref buffer, _dispatcher, out var message))
```

Comment:
Why do we need this?

Reply:
We chatted and we're going to check the cancellation inside HubConnectionContext and noop writes instead of using the token in multiple places